index.md

```diff
@@ -68,6 +68,6 @@ Instead of sending a call probe, or asking "hey, do you have time for a call?"
 
 This does three things: it time bounds the request, it gives the recipient some
 context, and it helps you remember what you wanted to talk about if the
-recipient can't or doesn't respond until later.
+recipient can't or doesn't respond until later. As a bonus, if they can't respond right away, they will have a better chance at suggesting who else you might ask in the meantime.
```

Comment:
Could you limit the line to 79 characters please, which makes it easier to read in plain text? Rest LGTM.